lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -48,6 +48,18 @@ sub _ofac_xml_zip {
     return _ofac_xml($output);
 }
 
+sub _validate_date {
+
+    my $file_date = shift;
+
+    # Date validation checker for both mm/dd/yyyy and dd/mm/yyyy
+    if ($file_date !~ /^(\d{1,2})\/(\d{1,2})\/(\d{4})$/) {
```

Comment:
These are usually easier to read with different delimiters.

I'd normally write this as:

```
return 1 if $file_date =~ m{^(\d{1,2})/(\d{1,2})/(\d{4})$};
return 0;
```